src/components/common/Dialogs/PermissionManagementDialog/PermissionManagementForm.tsx

```diff
@@ -72,31 +73,33 @@ const MSG = defineMessages({
 });
 
 interface Props extends ActionDialogProps {
-  close: (val: any) => void;
   users?: User[];
+  handleIsForceChange: SetStateFn;
```

Comment:
Recently I suggested we prefix the handlers that are passed as props with `on`: https://github.com/JoinColony/ColonyFrontEndLivingStandard/commit/4a76e00551428ec638b02efd342b24512bd541d7

Reply:
@jakubcolony The reason I used the name, `handleIsForceChange`, is that in total there are about 8 forms that use this same naming and wanted to be consistent for now. I suggest handling updating the naming for event handlers in a separate issue. Would that work?